servers/quarkus-server/src/main/java/org/projectnessie/server/config/VersionStoreConfig.java

```diff
@@ -25,10 +25,14 @@
 public interface VersionStoreConfig {
 
   @RegisterForReflection
-  public enum VersionStoreType {
+  enum VersionStoreType {
+    @Deprecated
+    IN_MEMORY,
```

Comment:
Effectively renaming the "old in-memory version-store" to `IN_MEMORY` and using `INMEMORY` for the new one.